test/functionalTest/cases/0947_v2_list_all_entities/GET_v2_entities.test

```diff
@@ -112,14 +112,12 @@ echo
 echo "05. POST /v2/entities with compound attribute value"
 echo "==================================================="
 payload='{
-      "id":   "E2",
-      "type": "T1",
-      "A0" : 
-        {
-          "value" : { "i1": "1", "s1": "s" }
-        }
-      
-
+  "id":   "E2",
```

Comment:
I know that "Now that I was there" feeling, in fact I sometimes also do it :)

However, please keep this to the minimum in future PRs. If you feel that something related with indentation needs to be changed, the use a PR for that, different for the one with functional changes. Otherwise, it is hard to do the review.

NTC